subsys/bluetooth/common/Kconfig

```diff
@@ -46,6 +46,12 @@ config BT_HCI_MESH_EXT
 	help
 	  Enable support for the Bluetooth Mesh HCI Commands.
 
+config BT_HCI_PROP_EXT
+	bool "Proprietary HCI Event support"
+	depends on BT_HCI_VS
```

Comment:
I'm not sure I'd make this dependency. AFAIK `BT_HCI_VS` is about the *Zephyr* vendor extensions, whereas someone may want to use a completely different set and register their own handler for them.